core/src/scene/importer.cpp

```diff
@@ -14,14 +14,29 @@ namespace Tangram {
 
 std::atomic_uint Importer::progressCounter(0);
 
-Node Importer::applySceneImports(const std::shared_ptr<Platform>& platform, const Url& scenePath,
-        const Url& resourceRoot) {
+bool isZipUrl(const Url& url) {
+    const char* extStr = ".zip";
+    const size_t extLength = strlen(extStr);
+    const size_t urlLength = url.string().length();
+    return (urlLength > extLength && (url.string().compare(urlLength - extLength, extLength, extStr) == 0));
```

Comment:
Implemented extension in the Url interface. a81f3d8.
Can you give this a look, I think I handled all the possible extension cases according to the Url spec.